test/unit/three_scale/middleware/handle_parse_error_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ThreeScale::Middleware::HandleParseErrorTest < ActionDispatch::IntegrationTest
```

Comment:
You do not need (but you can have ...) this integration test, as long you have the corresponding unit test
See:
https://github.com/3scale/porta/blob/3d07b9e15c22ae607dad7abc3aebca4bc0770e8a/test/unit/sidekiq_logging_middleware_test.rb
and
https://github.com/3scale/porta/blob/3d07b9e15c22ae607dad7abc3aebca4bc0770e8a/test/unit/three_scale/middleware/dev_domain_test.rb